packages/core/src/layout/Sidebar/Items.tsx

```diff
@@ -139,8 +139,7 @@ export const SidebarItem: FC<SidebarItemProps> = ({
       <NavLink
         className={clsx(classes.root, classes.closed)}
         activeClassName={classes.selected}
-        isActive={match => Boolean(match && !disableSelected)}
-        exact
+        end
```

Comment:
Do you know what's up with this API? Looks pretty weird :grin:

Reply:
Haha no idea. It wasn’t available until the latest alpha. Strange name.